build-system/cpp-cmake-conan/src/build.py

```diff
@@ -88,7 +88,7 @@ def build(
     host_arch: str,
     build_target: str,
     static_build: bool,
-    coverage: bool = True,
+    coverage: bool = False,
```

Comment:
default value is not really needed anymore since the argparser already stores default values

Reply:
Who knows where it's needed ;-)

Reply:
yep thats why I still approved it :)